src/components/Code.vue

```diff
@@ -24,7 +30,8 @@
       code: String,
       language: String,
       wasResponseAnError: Boolean,
-      statusText: String
+      statusText: String,
+      links: Array
```

Comment:
Code seems like too generic of a name now. Is there a better name to describe this?

Reply:
Is the problem really the name, or is the problem an SRP violation? Instead of changing the name, could this be solved by applying the OCP?

Reply:
I've created a wrapper component called `MainBody`, that contains the `Code` component and the links. I don't really like the name `MainBody`, but wasn't quite sure what else I could call it.